tests/layer_tests/onnx_tests/test_layernorm.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+import numpy as np
+import pytest
+from common.layer_test_class import check_ir_version
+from common.onnx_layer_test_class import OnnxRuntimeLayerTest
+
+from unit_tests.utils.graph import build_graph
+
+
+class TestLN(OnnxRuntimeLayerTest):
+    def create_net(self, shape, epsilon, axis, stash_type, ir_version):
+        """
+            ONNX net                   IR net
+            Input->LN->Output   =>    Input->Norm->Power
+        """
+
+        #
+        #   Create ONNX model
+        #
+
```

Comment:
useless comments - please remove it

Reply:
This comment is to align other op comment formats